app/views/ontologies/sections/_metadata.html.haml

```diff
@@ -41,7 +41,7 @@
           - c.row do
             = render FieldContainerComponent.new(label: t("ontologies.sections.label_groups")) do
               = horizontal_list_container(@ontology.group) do |v|
-                = render ChipButtonComponent.new(text: show_group_name(v), type: "static")
+                = render ChipButtonComponent.new(text: show_group_name(v), type: "static", tooltip: show_group_name(v))
```

Comment:
Fine by now. 

But the real, requirement was that, to be generalized to all the chips/links in the page, so that when the the text is too big, the tooltip will be shown (otherwise the tooltip is not needed, as the full text is already shown.